sms-android/snippet.html

```diff
@@ -0,0 +1,240 @@
+<div class="snippet" id="firefox-android-sms-{{ snippet_id }}">
+  <style text="text/css">
+   div.switch-{{ snippet_id }}#brandLogo {
+       background-size: {{ width|default(192, true) }}px {{ height ~ 'px' if height else 'auto' }};
+   }
+
+   .switch-{{ snippet_id }}#brandLogo {
+       height: {{ height|default(192, true) }}px;
+       width: {{ width|default(192, true) }}px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} {
+       background-color: white;
+       padding: 5px 20px;
+       border-radius: 5px;
+       box-shadow: 0 1px 3px rgba(72, 72, 72, 0.25);
+       font-family: "Open Sans", "Helvetica Neue", "Helvetica", "Arial", sans-serif;
+       font-size: 1em;
+       color: #484848;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} a {
+       color: #0073AA;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} p {
+       font-size: 1em;
+       display: block;
+       height: auto;
+       padding: 0;
+       width: auto;
+       margin: 10px 0 10px 0;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} #right-img {
+       float: right;
+       padding-left: 6px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} #text {
+       margin-top: 0px;
+       font-size: 1.05em;
+       line-height: 1.4;
+       margin-bottom: 20px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} h1 {
+       font-size: 1.4em;
+       margin-bottom: 0px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} .footer {
+       font-size: 0.78em;
+       margin: 0px;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} hr {
+       border-top: 1px dotted #c3c3c3;
+       height: 1px;
+       margin: 20px 0 20px 0;
+   }
+
+   #firefox-android-sms-{{ snippet_id }} .button, .button:link, .button:visited {
+       display: inline-block;
+       *display: inline;
+       *zoom: 1;
+       border-radius: 0.25em;
+       box-shadow: 0px 1px 0px 0px rgba(0, 0, 0, 0.2), inset 0px -1px 0px 0px rgba(0, 0, 0, 0.3);
+       background-color: #43a6e2;
+       background-color: #277ac1;
+       background-repeat: repeat-x;
+       background-image: -webkit-linear-gradient(top, #43a6e2, #277ac1);
+       /* Safari 5.1+, Chrome 10+ */
```

Comment:
We can't do that with the preview view because it goes away when a snippet is disabled. Also because it uses chrome:// urls for the images and some of the CSS. Also because the about:home code on the preview page is kind've out of date.

If they want to share the snippet on social channels they're going to have to stand up a special static version of it or we're going to have to resolve [bug 1084077](https://bugzilla.mozilla.org/show_bug.cgi?id=1084077).